lib/tasks/contributions.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+namespace :contributions do
+  desc 'Rerun serialisations for all published content'
+  task serialise: :environment do
+    count = 0
+    Contribution.published.select do |contribution|
```

Comment:
`Contribution.published.each` should do here as we are not passing any criteria to `select`